doc/dev/msgr2.rst

```diff
@@ -528,6 +533,62 @@ where epilogue is::
 
 late_status has the same meaning as in msgr2.1-crc mode.
 
+Compression
+-----------
+Compression handshake is implemened using msgr2 feature-based handshaking.
+In this phase, the peers exchange their supported compression methods and if compression is desired.  
```

Comment:
How about rephrasing this line? _the peers exchange (...) and if compression is desired_ is hard to grasp.